fast64_internal/oot/c_writer/oot_scene_writer/oot_trans_actor_to_c.py

```diff
@@ -0,0 +1,39 @@
+from ....utility import CData
+from ...oot_scene_room import OOTScene, OOTTransitionActor
+from ...oot_utility import indent
+
+
+def ootGetTransActorData(transActor: OOTTransitionActor):
+    """Returns the transition actors array's data"""
+    sides = [(transActor.frontRoom, transActor.frontCam), (transActor.backRoom, transActor.backCam)]
+    return (
+        "{ "
+        + (", ".join(["{ " + f"{room}, {cam}" + " }" for room, cam in sides]) + ", ")
```

Comment:
```suggestion
        + "{"
        + (", ".join(["{ " + f"{room}, {cam}" + " }" for room, cam in sides]) + ", ")
        + "}, "
```

Reply:
that part looks like this now: ``("{ { " + (", ".join([f"{room}, {cam}" for room, cam in sides]) + " }, "))`` (basically it's one line per argument), let me know if you still want this change 👍 (this file is called ``exporter/scene/to_c/transition_actor.py`` now)